pkg/odo/cli/registry/list.go

```diff
@@ -78,7 +78,9 @@ func (o *ListOptions) printRegistryList(w io.Writer, registryList *[]preference.
 		return
 	}
 
-	for _, registry := range *registryList {
+	regList := *registryList
```

Comment:
Can you add comment above the for loop explaining why we're looping backwards here?

Reply:
Will do.